packages/data-point/lib/reducer-list/factory.js

```diff
@@ -60,21 +75,19 @@ function parse (source) {
 module.exports.parse = parse
 
 /**
- * parses a raw transform
- * @param {string} source
- * @return {Transform}
+ * @param {Function} createReducer
+ * @param {Array} source
+ * @return {reducer}
  */
-function create (source = []) {
+function create (createReducer, source = []) {
```

Comment:
@acatl In my other PR, you commented that source should come first. Can you explain why you want that change?

Reply:
it is a factory method, its a common pattern for factory methods the first parameter to be input/source

Reply:
btw I had not noticed that `ReducerObject.create` had that order, I would really encourage the params to be the other way around

Reply:
Ok, I still don't understand why that pattern is better though. As I mentioned in the original thread, putting the data last seems better because if you want to use partial application, you would probably be more likely to want `_.partial(create, createReducer)` than `_.partial(create, source)`